src/Common/FunctionDocumentation.cpp

```diff
@@ -37,7 +146,11 @@ std::string FunctionDocumentation::syntaxAsString() const
 
 std::string FunctionDocumentation::returnedValueAsString() const
 {
-    return boost::algorithm::trim_copy(returned_value);
+    std::string result;
```

Comment:
Could write directly `std::string result = returned_value.description` instead of l. 149/150